### PR TITLE
Add /pods endpoint; response pod details

### PR DESCRIPTION
**Pull request by colin014**

### CODE REVIEW

File: client/api_clusters.go [hunk 1]
```diff
@@ -1001,6 +1001,125 @@ func (a *ClustersApiService) GetClusterStatus(ctx context.Context, orgId int32,
...
+	}
+
+	if localVarHttpResponse.StatusCode >= 300 {
+		newErr := GenericOpenAPIError{
+			body:  localVarBody,
+			error: localVarHttpResponse.Status,
+		}
+		if localVarHttpResponse.StatusCode == 200 {
+			var v PodDetailsResponse
+			err = a.client.decode(&v, localVarBody, localVarHttpResponse.Header.Get("Content-Type"))
```
Comment (stoader) [inline client/api_clusters.go]: The 

```err = a.client.decode(&v, localVarBody, localVarHttpResponse.Header.Get("Content-Type"))
 			if err != nil {
 				newErr.error = err.Error()
 				return localVarReturnValue, localVarHttpResponse, newErr
 			}
 			newErr.model = v
 			return localVarReturnValue, localVarHttpResponse, newErr``` seems to be similar across multiple if statements. This should be factored out into a separate function.

File: client/api_clusters.go [hunk 2]
```diff
@@ -1001,6 +1001,125 @@ func (a *ClustersApiService) GetClusterStatus(ctx context.Context, orgId int32,
...
+ClustersApiService Get pod details
+Getting pod details
+ * @param ctx context.Context - for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
+ * @param orgId Organization identification
+ * @param id Selected cluster identification (number)
+@return PodDetailsResponse
+*/
+func (a *ClustersApiService) GetPodDetails(ctx context.Context, orgId int32, id int32) (PodDetailsResponse, *http.Response, error) {
+	var (
+		localVarHttpMethod   = strings.ToUpper("Get")
```
[the remark this answers is not in the document]
Comment (colin014) [reply client/api_clusters.go]: This file is generated by `make generate-client`

File: client/api_clusters.go [hunk 3]
```diff
@@ -1001,6 +1001,125 @@ func (a *ClustersApiService) GetClusterStatus(ctx context.Context, orgId int32,
...
+		localVarPostBody     interface{}
+		localVarFormFileName string
+		localVarFileName     string
+		localVarFileBytes    []byte
+		localVarReturnValue  PodDetailsResponse
+	)
+
+	// create path and map variables
+	localVarPath := a.client.cfg.BasePath + "/api/v1/orgs/{orgId}/clusters/{id}/pods"
```
[the remark this answers is not in the document]
Comment (colin014) [reply client/api_clusters.go]: This file is generated by `make generate-client`

File: client/api_clusters.go [hunk 4]
```diff
@@ -1001,6 +1001,125 @@ func (a *ClustersApiService) GetClusterStatus(ctx context.Context, orgId int32,
...
+		return localVarReturnValue, localVarHttpResponse, err
+	}
+
+	localVarBody, err := ioutil.ReadAll(localVarHttpResponse.Body)
+	localVarHttpResponse.Body.Close()
+	if err != nil {
+		return localVarReturnValue, localVarHttpResponse, err
+	}
+
+	if localVarHttpResponse.StatusCode < 300 {
```
[the remark this answers is not in the document]
Comment (colin014) [reply client/api_clusters.go]: This file is generated by `make generate-client`

File: client/api_clusters.go (hunk 1 above)
Comment (colin014) [reply client/api_clusters.go]: This file is generated by `make generate-client`

**Pull request merged**